client/src/project/datasets/change/DatasetChange.present.js

```diff
@@ -103,7 +103,27 @@ function DatasetChange(props) {
     submitCallback={props.submitCallback}
     title={edit ? "Modify Dataset" : undefined}
     versionUrl={props.versionUrl}
+    toggleNewDataset={props.toggleNewDataset}
+    showAddDatasetOptions={!edit}
+    addDatasetOptionSelected={"addDataset"}
```

Comment:
```suggestion
    addDatasetOptionSelected="addDataset"
```
Not really something to fix, but I noticed you sometimes specify text in properties as `prop={"text"}` and sometimes `prop="text"`. It would make sense to keep it coherent when it's a fixed string